src/main/java/com/entropyteam/entropay/employees/services/HolidayService.java

```diff
@@ -51,4 +54,16 @@ public List<String> getColumnsForSearch() {
         return List.of("date");
     }
 
+    public List<Map<String, Object>> getHolidayYears() {
```

Comment:
I think here, we should change the return type, it should be List<Integer>, then in the FE parse the response ah we need

Reply:
nice, there I changed it both in Holiday Controller and in Holiday Service